<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>925</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="youLCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="digitCount">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>OPPONENT</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开始对战</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>新游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>载入残局</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存残局</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sueButton">
        <property name="font">
         <font>
          <pointsize>27</pointsize>
         </font>
        </property>
        <property name="text">
         <string>求和</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resignButton">
        <property name="font">
         <font>
          <pointsize>27</pointsize>
         </font>
        </property>
        <property name="text">
         <string>认输</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>ME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="meLCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="digitCount">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionkaishi"/>
    <addaction name="actionxin"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>残局</string>
    </property>
    <addaction name="actionzai"/>
    <addaction name="actionbao"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>动作</string>
    </property>
    <addaction name="actionrenshu"/>
    <addaction name="actionqiuhe"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionkaishi">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionxin">
   <property name="text">
    <string>新游戏</string>
   </property>
  </action>
  <action name="actionzai">
   <property name="text">
    <string>载入残局</string>
   </property>
  </action>
  <action name="actionbao">
   <property name="text">
    <string>保存残局</string>
   </property>
  </action>
  <action name="actionrenshu">
   <property name="text">
    <string>认输</string>
   </property>
  </action>
  <action name="actionqiuhe">
   <property name="text">
    <string>求和</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
